Dockerfile

```diff
@@ -1,5 +1,16 @@
-FROM node:8-onbuild
+FROM node:8.15.0-stretch
```

Comment:
Turns out [`onbuild`](https://hub.docker.com/_/node#nodeversion-onbuild) is deprecated and not recommended.